tools/bin/sdc-sbcreate

```diff
@@ -118,78 +122,83 @@ function grab_default
 function grab_sysinfo
 {
     echo "  > sysinfo"
-    sysinfo > ${TEMPDIR}/sysinfo.json
+    sysinfo > "${TEMPDIR}/sysinfo.json"
 }
 
 function grab_hw_config
 {
     echo "  > hardware configuration"
-    prtdiag -v > ${TEMPDIR}/prtdiag
-    prtconf -v > ${TEMPDIR}/prtconf
-    smbios -x > ${TEMPDIR}/smbios
-    psrinfo -v > ${TEMPDIR}/psrinfo
+    prtdiag -v > "${TEMPDIR}/prtdiag"
+    prtconf -v > "${TEMPDIR}/prtconf"
+    smbios -x > "${TEMPDIR}/smbios"
+    psrinfo -v > "${TEMPDIR}/psrinfo"
 }
 
 function grab_dlinfo
 {
     echo "  > datalink info"
-    dladm show-phys -m > ${TEMPDIR}/dladm.phys
-    dladm show-vnic > ${TEMPDIR}/dladm.vnic
-    dladm show-linkprop > ${TEMPDIR}/dladm.linkprop
-    dladm show-bridge > ${TEMPDIR}/dladm.bridge
+    dladm show-phys -m > "${TEMPDIR}/dladm.phys"
+    dladm show-vnic > "${TEMPDIR}/dladm.vnic"
+    dladm show-linkprop > "${TEMPDIR}/dladm.linkprop"
+    dladm show-bridge > "${TEMPDIR}/dladm.bridge"
 }
 
 function grab_ifconfig
 {
     echo "  > network interface info"
-    ifconfig -a > ${TEMPDIR}/ifconfig.a
+    ipadm show-addr > "${TEMPDIR}/ipadm.show-addr"
```

Comment:
There's stuff in ifconfig that isn't in ipadm (e.g. IFF flags).   If you want ipadm too, perhaps do that separately?

Reply:
Also, in general on ifconfig and routes, should you ALSO do this for each native or LX zone as well?

Reply:
Ok, I'll add that back in.

For zones...the config information is already captured in the vm json. I think this one might be time constrained, but I'll add it to the list.

Reply:
Less worried about zones, more worried about the ifconfig output.